src/test/java/com/hocs/test/glue/ukvi/MPAMCaseDetailsAccordionStepDefs.java

```diff
@@ -43,10 +43,6 @@ public void accordionInCaseDetailsContainsAllCorrectInformation(String accordion
                 accordionMPAM.openTriageAccordion();
                 accordionMPAM.assertAllTriageResponsesMatchInput();
                 break;
-            case "DRAFT":
-                accordionMPAM.openDraftAccordion();
-                accordionMPAM.assertAllDraftResponsesMatchInput();
-                break;
```

Comment:
Why do we no longer have a Draft accordion information check? has it been removed from the accordion?